recipes/r-abcrf/meta.yaml

```
@@ -0,0 +1,115 @@
+{% set version = '1.9' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-abcrf
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/abcrf_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/abcrf/abcrf_{{ version }}.tar.gz
+  sha256: b1eb36bd10dd71ede3274dcd243131b79fee32fb385c8b4d7715dcfe8c0ab022
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```

Workaround for Windows error (see https://github.com/conda/conda-build/pull/4786):

```suggestion
  missing_dso_whitelist:
     - '*/R.dll'            # [win]

```